nucypher/characters/lawful.py

```diff
@@ -424,6 +438,10 @@ def retrieve(self, message_kit, data_source, alice_verifying_key):
         cleartexts.append(delivered_cleartext)
         return cleartexts
 
+    def collect_evidence(self, capsule, cfrag, ursula):
+        from nucypher.policy.models import UnquestionableEvidence
+        return UnquestionableEvidence(capsule, cfrag, ursula)
```

Comment:
haha nice name

Reply:
Indeed!

Reply:
Haha, this was actually a bait so you NuCypherinos could help me with a hopefully better name. Although I've grown to like it :P